src/features/Apiexplorer/Schema/RecursiveContent/SchemaBodyHeader/index.tsx

```diff
@@ -116,6 +118,17 @@ const SchemaBodyHeader = ({
             ) : (
               <></>
             )}
+
+            {is_stream_types && (
+              <React.Fragment>
```

Comment:
do we need this React.fragment here? There is already a div wrapper?

Reply:
We are passing <React.Fragment> over div in some conditions in the code. Although we can remove it 👍